apps/web-giddh/src/app/add-company/add-company.component.ts

```diff
@@ -1159,6 +1161,7 @@ export class AddCompanyComponent implements OnInit, AfterViewInit, OnDestroy {
         this.socketCompanyRequest.utm_campaign = this.generalService.getUtmParameter('utm_campaign');
         this.socketCompanyRequest.utm_term = this.generalService.getUtmParameter('utm_term');
         this.socketCompanyRequest.utm_content = this.generalService.getUtmParameter('utm_content');
+        this.socketCompanyRequest.BusinessNature = this.secondStepForm.controls['otherBusinessNature'].value;
```

Comment:
if (this.secondStepForm.value.businessNature === "Other") {

Reply:
use this

Reply:
Done